### PR TITLE
Features/#66

### PR DESCRIPTION
**Pull request by DenisKulik**

### CODE REVIEW

**Review by VitaliKozhurov — APPROVED** [review]
Good job!

File: src/features/decks/ui/learnCard/LearnCard.tsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import { useState } from 'react'
+
+import s from './LearnCard.module.scss'
+
+import { TypographyVariant } from '@/common'
+import { Button, Card, Typography } from '@/components'
+import { RateLearnCard } from '@/features/decks/ui/learnCard/reteLearnCard'
```
Comment (vitalyabedik) [inline src/features/decks/ui/learnCard/LearnCard.tsx]: use relative import

File: vite.config.ts [hunk 2]
```diff
@@ -12,6 +12,7 @@ export default defineConfig({
       { find: '@components', replacement: path.resolve(__dirname, 'src/components') },
       { find: '@common', replacement: path.resolve(__dirname, 'src/common') },
       { find: '@assets', replacement: path.resolve(__dirname, 'src/assets') },
+      { find: '@features', replacement: path.resolve(__dirname, 'src/features') },
```
Comment (vitalyabedik) [inline vite.config.ts:15]: add alias to tsconfig.json

**Review by vitalyabedik — APPROVED** [review]
Well done!

**Pull request merged**